### ISSUE TITLE
The Month in WordPress: October 2022 Edition

### ISSUE THREAD

**Issue by rmartinezduque** [WordPress/Marketing-Team#131]
The estimated publication date for the October 2022 edition of The Month in WordPress is **November 10, 2022**.

### How can you help?

You can help by collecting any news or Make team updates that would be valuable and relevant to the post. Please **add them as a comment to this GitHub card with its relevant link and a brief description** by Wednesday, November 3, 2022, if possible. For example:

> The Training Team shared a new post on [how to become an online workshop facilitator or tutorial presenter](https://make.wordpress.org/training/2022/09/30/become-an-online-workshop-facilitator-or-tutorial-presenter-today/).

Alternatively, you can share them on the Make Marketing Slack channel and tag `Reyes Martínez` for awareness.

The draft will be added to this card and shared in the Marketing channel prior to publication. Your review will be appreciated, too. If you have any questions on how to contribute, please feel free to reach out.

### Additional background

The Month in WordPress is a monthly roundup of WordPress news and updates from the Make teams. It's usually published at the beginning of each month at WordPress.org/News, and it features news from the month prior to publication. To see its format and what's typically included, check out any of the [latest issues](https://wordpress.org/news/category/month-in-wordpress/).

**Comment by rmartinezduque** [comment]
Hi! In order to facilitate the content curation for the Month in WordPress, I'd welcome help in collecting some updates that may be relevant to be featured in the next edition

- Announcements or significant news from Make WP teams
- Major updates or news from flagship WordCamps (WordCamp US, WordCamp Asia, and WordCamp Europe)
- Testing/feedback calls that may be relevant to be featured in the next edition

### How can you help collect links/updates?

Edit the GitHub comment below and share any links (with a short description) under their relevant category. Alternatively, if it's easier for you, you can directly post a new comment with the links you want to suggest.

**Please note that, for brevity, it may not be possible to include all links/updates in the final post**. If you have any questions or are unsure how to help, please feel free to ask.

**Comment by rmartinezduque** [comment]
⬇️ _**Edit this GitHub comment to add new links to the list. Don't forget to include your WordPress ID/username next to the update so that we can credit all contributions**_. See the examples below. Please don't include meeting notes, agendas, chat summaries or recaps, etc. ⬇️

### Community announcements and newsletters
- October 2022 Meetup Organizer Newsletter: 

### Polyglots announcements and newsletters
- October 2022 Polyglots Newsletter: 

### WPDiversity upcoming events or reports

- In September 2022, the [#WPDiversity](https://make.wordpress.org/community/tag/wpdiversity/) group successfully held a [speaker training workshop to empower WordPress women](https://make.wordpress.org/community/2022/10/17/report-diverse-speaker-workshop-september-2022/) in India for two days. - Added by `rmartinezduque`
- The list of upcoming #WPDiversity events can be found in the [Community Handbook](https://make.wordpress.org/community/handbook/meetup-organizer/event-formats/diversity-speaker-training-workshop/#upcoming-wpdiversity-workshops). - Added by `rmartinezduque`

### LearnWP news or roundup post

- The first step-by-step tutorial on building a Gutenberg block, [Introduction to Block Development: Build your first custom block](https://learn.wordpress.org/course/introduction-to-block-development-build-your-first-custom-block/), is now live on Learn WordPress. - Added by `rmartinezduque`

### Feedback & testing requests
- [The Plugin Dependencies feature plugin](https://wordpress.org/plugins/wp-plugin-dependencies/) is available for testing. Feedback should be provided by December 1, 2022. - Added by `rmartinezduque`

### Other Make Updates (Design, Plugins, Documentation, Marketing, etc)
-

**Comment by nalininsbs** [comment]
@abhansnuk dictation is not working with the edit. Thank you for the help to test. Can you try and create an image if you can.

**Comment by rmartinezduque** [comment]
@nalininsbs @abhansnuk, thanks for bringing this out. I have taken the following screenshot that shows where to edit the comment:

![screenshot](https://user-images.githubusercontent.com/20700757/196881187-2ece784d-99f6-45f6-be85-a436fa3e42f1.png)

Please also find below more instructions:

- In the upper-right corner of the comment, click the three-dot menu icon, then click `Edit`.
- Add the content you'd like to include in the comment window and, once you have finished, click `Update comment`.

Let me know if that helps. Another alternative to facilitate this task is to create a Google document where we can add suggestions. Do you think that would work better for everyone?

**Comment by abhansnuk** [comment]
Thanks @rmartinezduque. The difficulty was editing with dictation tools and then saving the change. 

In general terms without accessibility tools, I think there might be a danger of other content being deleted accidentally. We might be able to use one of the newer GitHub beta tools instead of Google doc that @vimes1984 and I had been trying. I will try a test and see if that could help in the future, and send the test to you to see what you think. Thanks

**Comment by murillol** [comment]
@rmartinezduque I know the deadline for this is today November 8, but can we allow people to send items in until tomorrow November 9 (and mention it in the meeting)? Thanks!

**Comment by rmartinezduque** [comment]
Yes, sure @murillol. Thanks for asking! I think until Friday (Nov 11) it's totally fine to send items or suggestions.

**Comment by rmartinezduque** [comment]
**Update**: I'm hoping to finish the draft for October's Month in WordPress today or tomorrow. In order to allow some time for review/feedback, the expected publishing date will be Friday (Nov 18) or Monday, November 21.

**Comment by rmartinezduque** [comment]
The draft for the October 2022 edition of The Month in WordPress is ready for review. I'm hoping to publish it early next week. If you have any suggestions, **please share them before the end of the day tomorrow (October 17)**.

Link: https://docs.google.com/document/d/1sRMrJ5wEQ0afouoQzgDc2MDL3m7-JEqA9P4BAEzoC24/edit?usp=sharing

Also, remember to include your WordPress.org profile link or username as a comment at the end of the document so that all contributions can be credited. Thank you!

**Comment by rmartinezduque** [comment]
Update: I just updated the October 2022 edition of The Month in WP with the recent news of State of the Word: https://docs.google.com/document/d/1sRMrJ5wEQ0afouoQzgDc2MDL3m7-JEqA9P4BAEzoC24/edit#. I'm hoping to publish it tomorrow morning (CET time).

**Comment by rmartinezduque** [comment]
**The Month in WordPress – October 2022 is now published**: https://wordpress.org/news/2022/11/the-month-in-wordpress-october-2022/. Thank you to everyone who contributed to this edition!

If you'd like to contribute to the next edition of the Month in WordPress, feel free to express your interest and subscribe to this GitHub card: https://github.com/WordPress/Marketing-Team/issues/142.
